Gordon360/Models/CCT/Context/CCTContextProcedures.cs

```diff
@@ -639,7 +639,7 @@ public virtual async Task<int> CREATE_RIDEAsync(string RIDEID, string DESTINATIO
             return _;
         }
 
-        public virtual async Task<int> CREATE_SOCIAL_LINKSAsync(string USERNAME, string FACEBOOK, string TWITTER, string INSTAGRAM, string LINKEDIN, string HANDSHAKE, OutputParameter<int> returnValue = null, CancellationToken cancellationToken = default)
+        public virtual async Task<int> CREATE_SOCIAL_LINKSAsync(string USERNAME, string FACEBOOK, string TWITTER, string INSTAGRAM, string LINKEDIN, string HANDSHAKE, string CALENDAR, OutputParameter<int> returnValue = null, CancellationToken cancellationToken = default)
```

Comment:
@andrew-wzj it looks like you're still updating this code manually, but that won't work because this is just a representation of the database stored procedure, so it will break when the code tries to call the stored procedure.

You need to actually update the `CCT.dbo.CREATE_SOCIAL_LINKS` stored procedure in the database, and then you can do an EFPT refresh and all this code will be updated automatically for you.